web/app/components/document/sidebar.js

```diff
@@ -43,13 +42,28 @@ export default class DocumentSidebar extends Component {
   @tracked title = this.args.document.title || "";
   @tracked summary = this.args.document.summary || "";
   @tracked tags = this.args.document.tags || [];
-
   @tracked contributors = this.args.document.contributors || [];
   @tracked approvers = this.args.document.approvers || [];
+  @tracked product = this.args.document.product || "";
 
   @tracked userHasScrolled = false;
   @tracked body = null;
 
+  /**
+   * If the user is saving a new product, this will be the new product.
+   * Reset when the save task fails.
+   */
+  @tracked _newProduct = null;
```

Comment:
Curious why we couldn't just use `this.product` for the places where we're using this?

Reply:
Good call. Updated https://github.com/hashicorp-forge/hermes/pull/109/commits/41221b5abf09005f0e7362bd0e52ed5db1e87850